ega-data-api-dataedge/src/main/resources/application.yml

```diff
@@ -1,15 +1,29 @@
-spring.profiles: no-oss
+spring.profiles.active: no-oss,logger-log,add-user-ip-headers
 
 eureka.client.enabled: false
 
+management.server.port: -1
```

Comment:
I though this disables completely all the actuator endpoints via http and they are just available by jmx. Is this intended?

Reply:
yes, followed https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-monitoring.html#production-ready-disabling-http-endpoints, i think we only need `health` and `info` otherwise we would have to protect with some form of auth the other endpoints I don't think it is worth he hassle and they should be available over HTTP, as I don't see a use case for them, besides management.

Reply:
but maybe I a missing a use case where e.g. `restart` or any other similar management endpoint would be needed in a production service running in a container. Do you have any experience ?

Reply:
I can also add `management.endpoints.jmx.exposure.exclude=*`

Reply:
The endpoints that I have used in prod are our /info (is the same concept but with our implementation because it did not exist at the time), the logging one to change the log level (again our implementation because spring did not have one), the health endpoint and we have a custom pause/resume one which implementation depends on the service and is usually not implemented but I don't think spring implements anything similar to this (just the shutdown). I agree that the metrics maybe would be useful but we can enable as needed as you said

management.server.port does not work. In this version of Spring is management.port but that completely disables the http endpoints so that after enabling them later they do not work. So i'd delete this line.

There have been a lot of changes in the properties names between 1.4 and 2.1. The link for 1.4 is https://docs.spring.io/spring-boot/docs/1.4.7.RELEASE/reference/htmlsingle/#production-ready-disabling-http-endpoints

management.endpoints.jmx.exposure.exclude=*
this endpoint does not work on 1.4. If you want to disable jmx you can use spring.jmx.enabled=false

Reply:
ok, will change to that.